Source/WPEFramework/PluginServer.cpp

```diff
@@ -840,13 +841,19 @@ namespace PluginHost
 
         if (children.Count() > 0) {
 
+            // make sure to wakeup PIDs in opposite order to hibernation
+            // to quickly go over not hibernated PIDs and abort currently processed
+            std::list<ThreadId> childrenBackward;
```

Comment:
Do we really need to crete a copy of the list? can yu not ->Reset(false (or ->Count()) ? set the childre interator to the end, and than while(children.Previous()) run throughthe list and WakeUp....?